doc/tutorial/mutli_arch.hpp

```diff
@@ -0,0 +1,158 @@
+#error This file is for documentation only - DO NOT INCLUDE
+/**
+
+@page multiarch Handling Multiple Architecture Targets
+
+# The Issue
+The **EVE** library uses compile-time code generation process to ensure a high-level of performance.
+However, there can be many reasons why your application or library maybe run on unknown systems.
+Thus, you may want to be able to write your SIMD kernel once but be able to adapt to your end-user's
+hardware.
+
+A first solution could be to just recompile a given kernel with different architecture settings and
+link everything. Then, at runtime, use eve::is_supported to choose the correct implementation
+between the available ones. If the premise of this idea is correct, you'll face issues as the
+core of **EVE** is based on templated callable object, you may end up violating the One Definition
+Rule and ends up with a binary containing the incorrect symbols and implementations.
+
+# From static to dynamic dispatch
+The correct process is based on the same premisses but avoids those pitfalls. The idea will be to
+isolate the various version of a given kernel into separate binaries. In this case, no multiple
+definition can happen. This means using **dynamic libraries** to store your code and use
+**dynamic loading**.
```

Comment:
I would assume if a person does SIMD they have a vague idea of what a dynamic library is.

Reply:
Let be precautious